composables/drop/massmint/useDropMassMint.ts

```diff
@@ -90,12 +90,17 @@ export default () => {
 
       loading.value = true
 
-      const previewItems = single
-        ? ([previewItem.value] as GenerativePreviewItem[])
-        : generateMassPreview(
-            amountToMint.value,
-            mintedAmountForCurrentUser.value,
-          )
+      const previewItems = (
+        single
+          ? [previewItem.value]
+          : [
+              previewItem.value,
+              ...generateMassPreview(
+                amountToMint.value - 1,
+                mintedAmountForCurrentUser.value,
```

Comment:
![CleanShot 2024-04-05 at 09 31 43@2x](https://github.com/kodadot/nft-gallery/assets/44554284/c2d64f75-bc2a-4c40-81fb-1f96221dd24b)

untested but a `+ 1` should make it work correctly :d 

```suggestion
                mintedAmountForCurrentUser.value + 1,
```

Reply:
Thanks!